### ISSUE TITLE
select multiple items with mouse?

### ISSUE THREAD

**Issue by mg1075** [lou/multi-select#183]
If this works like a standard html `<select>` with the `multiple` attribute, 
shouldn't there be the ability to select multiple elements with the mouse
instead of just one at a time?

**Comment by lou** [comment]
You are right :smile: I never thought of this one.

But what behavior do you expect on this ? I do not find any good UX.

The user select many rows and
  - click another time on the selection to select it.
  - it automatically select the items on selection end.

In both case I find it weird.

**Comment by sgtbigman** [comment]
Maybe once the 'Ctrl' or 'Command' key is 'let go of' (keyup), all the clicked/selected items would be added.

Anyways, just a thought.

**Comment by ksodari** [comment]
Yes as mentioned by @sgtbigman allowing user to press 'CTRL' key and select multiple values and then click on a button which does the function of moving selected items from selectable side to selection side..
